net.sourceforge.DuneLegacy.yaml

```diff
@@ -0,0 +1,35 @@
+app-id: net.sourceforge.DuneLegacy
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: dunelegacy
+rename-icon: dunelegacy
+rename-desktop-file: dunelegacy.desktop
+rename-appdata-file: dunelegacy.appdata.xml
+finish-args:
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+
+modules:
+  - name: dunelegacy
+    buildsystem: autotools
+    post-install:
+      - install -Dm 0644 dunelegacy.desktop ${FLATPAK_DEST}/share/applications/dunelegacy.desktop
+      - install -Dm 0644 dunelegacy.appdata.xml ${FLATPAK_DEST}/share/appdata/dunelegacy.appdata.xml
+      - install -Dm 0644 dunelegacy-128x128.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/dunelegacy.png
+    sources:
+      - type: git
+        url: https://git.code.sf.net/p/dunelegacy/code
```

Comment:
Please configure https://github.com/flathub/flatpak-external-data-checker/ to automate updates.

Reply:
Added